Automattic/jetpack
# AI Assistant: Separate rules prompt message and refactor title props check

Fixes #
This PR is aimed at making the responses better, with less "Sure! Here it is" kind of answers. It also improves the title handling, disabling the Markdown rule for that case.

## Proposed changes:

* IMPORTANT: Changes the AI context
* Separates the rules from the `system` context message to a separate `user` message
* Removes `isGeneratingTitle` prop
* Refer to `const`s instead of strings when dealing with prompt types
* Tweaks the prompt/rules

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Use the AI Assistant block and check all cases. It should work as normal with better results
